packages/block-editor/src/components/border-radius-control/input-controls.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * External dependencies
+ */
+import { noop } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { __experimentalUnitControl as UnitControl } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+const CORNERS = {
+	topLeft: __( 'Top left' ),
+	topRight: __( 'Top right' ),
+	bottomLeft: __( 'Bottom left' ),
+	bottomRight: __( 'Bottom right' ),
+};
+
+export default function BoxInputControls( {
+	onChange = noop,
+	values: valuesProp,
+	...props
+} ) {
+	const createHandleOnChange = ( corner ) => ( next ) => {
+		onChange( {
```

Comment:
Could we avoid importing `noop` and bail early in this function if `! onChange`?

Reply:
✅ Done.